src/main/webapp/script.js

```diff
@@ -367,17 +371,67 @@ function getFailedJobsCost(aggregated) {
         failedCost += aggregated[category][costs].price;
       }
     }
-    jobData.push(failedCost);
-    data.push(jobData);
-    isAllZero = isAllZero && ((failedCost == 0) || isNaN(failedCost));
+    if (failedCost != 0) {
```

Comment:
You shoud use !== instead of !=. See [link](https://stackoverflow.com/questions/11960603/difference-between-and) for more information and for some examples.

Reply:
Resolved in 953712c.